charts/umbrella/values.yaml

```diff
@@ -579,6 +579,15 @@ bpndiscovery:
       endpointAddress: /bpndiscovery
       documentation: /bpndiscovery/swagger-ui/index.html
       timeToLive: "31536000"
+    discoveryfinderClient:
+      baseUrl: "semantics.tx.test/discoveryfinder"
+      registration:
+        clientId: sa-cl22-01
+        clientSecret: "client-secret"
+        authorizationGrantType: changeme
```

Comment:
Hi @tunacicek sorry I didn't notice your comment from yesterdays.

That looks a bit like a mix up.

And just to know: where is sa-cl21-01 used? should that be configured somewhere as well?